proposals/instructor-training-selection.md

```diff
@@ -0,0 +1,146 @@
+# Instructor Training Selection
+
+**Warning: This is only a draft.**
+
+**Warning: This only applies for the two-day training course in December 2015.**
+
+## Terminology
+
+-   An **instructor** is someone who teaches a workshop.
+    Instructor training (IT) prepares them to do this.
+
+## Why Select Attends for Instructor Training
+
+SCF wants to train as many instructors as possible
+but demand for instructor training is higher than what SCF is able to offer
+at the moment.
+
+## Eligibility
+
+We expect all students to gather in groups
+that are able to co-locate in one room for the duration of the Instructor Training.
+Each group should commit to jointly run one workshop no later than April 1st, 2016.
+**Please note: how to run a workshop is covered in the curriculum.**
+
+### Members of the groups
+
+The members of the group should be on the current waiting list
+or
+demonstrate sufficient competence with core tools used in the lessons they will teach.
+
+Each students is expected to:
+
+-   Watch one of the two videos below
+    and submit a paragraph of feedback on the video you chose to watch
+    as part of your application.
+
+    - https://vimeo.com/139316669
+    - https://vimeo.com/139181120
+
+-   Submit a pull request to the GitHub repository at
+    https://github.com/swcarpentry/training-prep-2015-09.
```

Comment:
Only if they're intending to teach Software Carpentry - if they're aiming for Data Carpentry, no PR is required. (Which makes me think we should drop the requirement for Software Carpentry as well to avoid confusion?) (We should also check with @tracykteal to see if we can put in some other task for DC would-be instructors.)